src/olympia/zadmin/tests/test_views.py

```diff
@@ -50,20 +50,32 @@
     'zadmin', 'tests', 'resources')
 
 
-class TestHome(TestCase):
+class TestHomeAndIndex(TestCase):
     fixtures = ['base/users']
 
     def setUp(self):
-        super(TestHome, self).setUp()
+        super(TestHomeAndIndex, self).setUp()
         self.client.login(email='admin@mozilla.com')
 
-    def test_get(self):
+    def test_get_home(self):
         url = reverse('zadmin.home')
         response = self.client.get(url, follow=True)
         assert response.status_code == 200
         assert response.context['user'].username == 'admin'
         assert response.context['user'].email == 'admin@mozilla.com'
 
+    def test_get_index(self):
+        # Add fake log that would be shown in the index page.
+        user = UserProfile.objects.get(email='admin@mozilla.com')
+        core.set_user(user)
```

Comment:
Do you still have to do this when you're passing user in as an arg to `ActivityLog.create`?